### ISSUE TITLE
Overlay behavior

### ISSUE THREAD

**Issue by maethu** [4teamwork/ftw.referencewidget#4]
- Background should not be scrollable
- Some kind of background fadeout
- Close with `ESC` - only close this window not others.

**Comment by jone** [comment]
- close overlay when clicking on the background

**Comment by maethu** [comment]
@jone On nested overlayes its not that easy... even more if the don't know each other. 

Example:
1. Overlaye (Edit Overlay) -> Register keyup event for ESC to close the overlay
2. Overlay (Refbrowser) -> Register keyup event for ESC to close the overlay.

So on ESC both overlays are closing... Since the refbrowser don't know anything about the simplelayout overlay and vice versa. It's also possible that the refbrowser overlay is not nested. 

I guess we're no going to solve this problem, without a centralised overlay system  :-( Which I don't want...

**Comment by jone** [comment]
But this does not influence the close-on-click feature, since the click-handler can explicitly be registered on the background/fadeout `<div>`, which then will be "nestable".

On esc:
I'm not too familiar with the event handling mechanism, but in jQuery you can do a `.one`, which unregisters itself. When the handler prevents executing later handlers (something like `e.stopPropagation`??) and the last registered handler is called first, it should "jus work" when using `.one`, right?

**Comment by maethu** [comment]
Yep... the closing the Overlay by clinking somewhere outside of the overlay should be easy.


Nope. It's not that easy, since all `keydown` events are registered on the `document` you cannot really control the order in which the events are fired.
